scripts/scil_compute_fodf_metrics.py

```diff
@@ -64,6 +64,11 @@ def _build_arg_parser():
     p.add_argument('--rt', dest='r_threshold', type=float, default='0.1',
                    help='Relative threshold on fODF amplitude in percentage  '
                         '[%(default)s].')
+    p.add_argument('--absolute_peak_values', action='store_true',
+                   help='If set, the peak_values are not normalized for each '
+                        'voxel, \nbut rather they keep the actual fODF '
+                        'amplitude of the peaks. \nAs a result, the peaks are '
```

Comment:
I bugged a bit on the last line. "Also, peaks are given as unit directions instead of being proportional to peak_values." or something?